DEX.md

```diff
@@ -0,0 +1,63 @@
+# What is a DEX?
+
+A Decentralized EXchange (DEX) is a smart contract that enables users to trade tokens without relying on a centralized intermediary. Instead, trades are executed directly on-chain through liquidity pools.
+
+# Liquidity Pool:
+
+Each DEX consists of one or more liquidity pools. A pool typically holds two tokens (say, $T_0$ and $T_1$). 
+In an ideal world, there would be a pool for every possible token pair.
+Role of the users interacting with a liquidity pool can be:
+
+1. Liquidity Providers (LPs): Users who deposit tokens into pools are called liquidity providers.
+2. Traders: Users who swap one token for another. The exchange rate is determined by the ratio of tokens in the pool.
+3. Platform providers: Owners of the DEX.
+
+
+When a trade occurs, the trader pays a fee. This fee usually consists of:
+1. A platform fee that goes to the DEX protocol, and
+2. An LP fee that is distributed to liquidity providers, proportional to the amount of liquidity they contributed.
+
+Some platforms allow LPs to claim and reinvest their fees, effectively compounding their rewards.
+
+# Market Ratio:
+
+A liquidity pool maintains a market ratio (MR) between its two tokens:
```

Comment:
what does this mean intuitively? something like the number of one tokens must at least be half as much and at most be twice as much as the number of the other token? 
This is also basically the "price"? 

Just thought adding such an intuition might help to then parse the formula more easily